plugins/notification-resources/src/components/MessagesPreview.svelte

```diff
@@ -0,0 +1,69 @@
+<!--
```

Comment:
We need to move this module to chunter and use some kind of extension.

Reply:
I've used `view.mixin.Preview` in `chunter` model and moved this component to `chunter-resources`, thanks

Reply:
It is still in notification-resources

Reply:
Sorry, I didn't notice the comment was related to `MessagesPreview`. Moved both of the files to `chunter` plugin and used new mixin `NotificationPreview` (as `PreviewPresenter` didn't play well with issues). Thanks!